website/docs/components/alert/partials/code/component-api.md

```diff
@@ -24,10 +24,10 @@ Title, description, actions, and generic content are passed into the alert as yi
 
 <Doc::ComponentApi as |C|>
   <C.Property @name="<[A].Title>" @type="yielded component">
-    A container that yields its content inside the `"title"` block. Content inherits its style.<br/><br/>`...attributes` spreading is supported.
+    A container that yields its content inside the `"title"` block. Content inherits its style.<br/><br/>This component supports use of [`...attributes`](https://guides.emberjs.com/release/in-depth-topics/patterns-for-components/#toc_attribute-ordering).
```

Comment:
Nit: Instead of including double break tags, would it work to just separate the sentences by a blank line so that separate paragraphs are generated?

Reply:
Unfortunately, break tags are required to force new paragraphs within the Component API component.

Reply:
cc @jorytindall ☝🏻